static/js/user_profile.js

```diff
@@ -184,9 +184,13 @@ export function show_user_profile(user) {
     $("#user-profile-modal-holder").html(render_user_profile_modal(args));
     overlays.open_modal("user-profile-modal", {autoremove: true});
     $(".tabcontent").hide();
-    $("#profile-tab").show(); // Show general profile details by default.
+    if (default_tab === 1) {
+        $("#user-profile-streams-tab").show();
+    } else {
+        $("#profile-tab").show(); // Show general profile details by default.
+    }
     const opts = {
-        selected: 0,
+        selected: default_tab,
         child_wants_focus: true,
```

Comment:
Relardly, do the calls to `settings_account.initialize_custom_user_type_fields`, below, need to be moved inside the `callback`? I don't think we want to be doing that initialization work when not visiting the target tab.

Reply:
Added new commit for that 76863830be457d2a915c1246933e3f6ccd172b90.